app/src/main/kotlin/com/wire/android/ui/home/conversationslist/common/CustomGroupAvatarDrawing.kt

```diff
@@ -0,0 +1,241 @@
+/*
+ * Wire
+ * Copyright (C) 2025 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ */
+package com.wire.android.ui.home.conversationslist.common
+
+import androidx.compose.foundation.Canvas
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.size
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Path
+import androidx.compose.ui.unit.dp
+import com.wire.android.util.ui.PreviewMultipleThemes
+
+@Suppress("ArgumentListWrapping")
+@Composable
+fun CustomGroupAvatarDrawing(
```

Comment:
Might be annoying to maintain if the design changes, but TBH I can't find anything simpler without adding a 3rd party library just for these icons.

For the sake of documentation... how did you "convert" the paths in the SVG to the canvas code below?
Just reading it and rewriting in order?

If there was any tool or anything like that, it would be nice to document in code, in case someone ever needs to change it.